File: WorldFeed/Server/src/Services/Upload/WorldFeed.Upload.API/DTO/UploadedImageDetailsDTO.cs
﻿namespace WorldFeed.Upload.DTO
{
    using Newtonsoft.Json;

    public class UploadedImageDetailsDTO : IUploadedImageDetails
    {
        [JsonProperty("w")]
        public int? Width { get; set; }

        [JsonProperty("h")]
        public int? Height { get; set; }

        [JsonProperty("image_type")]
        public string ImageType { get; set; }
    }
}

File: Sharebook/Client/src/app/core/Core/DTO/UploadedImageDetailsDTO.ts
import {IUploadedImageDetails} from "../../Public/Models/Interfaces/DTO/IUploadedImageDetails";

export class UploadedImageDetailsDTO implements IUploadedImageDetails {
  // [JsonProperty("w")]
  public width?: number;

  // [JsonProperty("h")]
  public height?: number;

  // [JsonProperty("image_type")]
  public imageType: string;
}
